### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Bluntvloei** [RDG88/lovelace-awtrix-light-display-card#2]
Can you add tap actions to the lovelace card? i would like to navigate (open a bubblecard) when clicking on the clock screen

**Comment by RDG88** [comment]
Will do!

**Comment by Bluntvloei** [comment]
> Will do!

Thank you 😁

**Comment by Knight1981** [comment]
Yeah, tap actions would be really cool. I would love it to click on the card to trigger the automation that refreshes the display automatically in the background. :-)

Thanks a lot for your awesome work!
